pkg/authorizer/metadata_authorizer.go

```diff
@@ -77,7 +77,8 @@ func (s MetadataBasedAuthorizer) GetMatchingDbRole(ctx context.Context, tableNam
 }
 
 func (s MetadataBasedAuthorizer) Configure(_ string, _ map[string]dbrole.DbRole) {
-	// TODO - Set "service role" to DB role mapping here
+	// TODO - Set "service role" to DB role mapping here"
+	// Currently MetadataBasedAuthorizer, doesn't support service to DB role mapping
```

Comment:
@krishnamiriyala, how correct is the statement that `MetadataBasedAuthorizer` "doesn't support service to DB role mapping", considering that in `GetMatchingDbRole()`, we are actually extracting a **_role_** key-value pair and mapping the value to a DB role? What do you think about changing the comment to something like "Currently MetadataBasedAuthorizer, doesn't support **custom** service to DB role mapping"?